crates/sel4-capdl-initializer/core/src/lib.rs

```diff
@@ -359,6 +359,28 @@ impl<'a, N: ObjectName, D: Content, M: GetEmbeddedFrame, B: BorrowMut<[PerObject
                                 }
                             }
                         }
+                        #[sel4_cfg(any(ARCH_IA32, ARCH_X86_64))]
+                        Object::IrqMsi(obj) => {
+                            init_thread::slot::IRQ_CONTROL.cap().irq_control_get_msi(
+                                obj.extra.pci_bus,
+                                obj.extra.pci_dev,
+                                obj.extra.pci_func,
+                                obj.extra.handle,
+                                *irq,
+                                &cslot_to_relative_cptr(slot),
+                            )?;
+                        }
+                        #[sel4_cfg(any(ARCH_IA32, ARCH_X86_64))]
```

Comment:
Did this because `#[sel4_cfg(ARCH_X86_64)]` doesn't work currently

Reply:
Sorry, I made a typo in my comment. I meant that it would be better to do a generic x86 (by checking `CONFIG_ARCH_X86`) check but that doesn't work at this moment due to https://github.com/seL4/rust-sel4/blob/d3341eacd9da15c2edf96ac2c05afbb67506152b/crates/sel4/src/arch/mod.rs#L10 I think.